### ISSUE TITLE
$.isNaN() undocumented

### ISSUE THREAD

**Issue by dmethvin** [couchand/Count-Form-u-la#5]
Looks like at least some of the unit tests  use jQuery.isNaN() which is undocumented and will be removed in 1.7.

**Comment by couchand** [comment]
Where the heck did I pick that up?  Hmmm.  Thanks for pointing that out.

Ok, do you want collaborator access or should I take care of it?  If the latter, what would you suggest for a fix?  Regular JavaScript isNaN()?

**Comment by dmethvin** [comment]
Mainly it is just a courtesy call to make sure it didn't take you by surprise. In 1.7 we're putting in a documented jQuery.isNumeric that you could use (note that it's the opposite of isNaN) but of course you'd have to test to see which is supported. I would just create  your own implementation if you need it, it's just one line.

```
isNumeric: function( obj ) {
    return obj != null && rdigit.test( obj ) && !isNaN( obj );
},
```

**Comment by couchand** [comment]
Cheers!  After looking at the test in question, I realized that I actually did want regular JS isNaN.

fixed in 92127a0a05362ff215226f009e44c05c60a96a34

**Comment by couchand** [comment]
also fix use in widget
